markdown/events/2022/hackathon-march-2022.md

```diff
@@ -11,14 +11,16 @@ import_typeform: true
 ---
 
 
-Please join us for the March 2022 nf-core hackathon! 
+# Welcome
 
-### Topic: Documentation
-
-As a result of the fun changes introduced to pipelines with Nexflow DSL2, the focus of this hackathon will be all things documentation. This will include documentation for pipelines and modules, training, tutorials, nf-core tools etc.
+Please join us for the March 2022 nf-core hackathon!
+The theme for this hackathon will be _Documentation_.
```

Comment:
Interesting that the markdown linting didn't complain here this time...

Reply:
It's the _consistency_ within a file I think. So either use single underscores or asterisks, but not both.

I think it could be nice to standardise to use single underscores for italics and double asterisks for bold in all pages to be honest. But not important.